coverage/clover.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
- [ ] [OPTIONAL] you guys did a great job 👏 I would suggest adding the coverage folder inside the .gitignore and removing it from the repo since it has a lot of unnecessary files and also it will be auto-generated every time user run the tests

Reply:
Hi @ichala 👋, this is a valuable Note and I will do so.

Thank you very much 👏👏.